src/Microsoft.Identity.Web/WebAppAuthenticationBuilderExtensions.cs

```diff
@@ -79,7 +80,13 @@ public static class WebAppAuthenticationBuilderExtensions
             var b2cOidcHandlers = new AzureADB2COpenIDConnectEventHandlers(openIdConnectScheme, microsoftIdentityOptions);
 
             builder.Services.AddSingleton<IOpenIdConnectMiddlewareDiagnostics, OpenIdConnectMiddlewareDiagnostics>();
-            builder.AddCookie(cookieScheme);
+            builder.AddCookie(cookieScheme, options =>
```

Comment:
Option 1.
If the path is not set, we set it to default one of our choice.
**Pro** The setup is all in one place and the scheme we use is abstracted from the user.
**Con** If the user doesn't call `AddMicrosoftIdentityUI` and wants to use the default ASP.NET Core path (/Account/AccessDenied), we'll end up replacing it. 
**Con** We're kind of mixing non-UI logic with UI logic.